### ISSUE TITLE
Code Block Highlight not working (Invalid syntax highlight definitions: define at least one highlight.js language mapping)

### ISSUE THREAD

**Issue by nicolaszu** [ueberdosis/tiptap#742]
**Problem**
I'm getting the following error when recreating the [code block highlight example](https://github.com/scrumpy/tiptap/blob/master/examples/Components/Routes/CodeHighlighting/index.vue):

`Invalid syntax highlight definitions: define at least one highlight.js language mapping. 
`
I've installed highlight.js, imported it and passed on the languages object as described in the example.

I'm unsure as to how to debug this unfortunately. I've tried a few different things but can't seem to get it working. Any help would be appreciated!

**Screenshots**
![image](https://user-images.githubusercontent.com/31318730/85798351-51049700-b70b-11ea-962c-f1aea14fe968.png)
![image](https://user-images.githubusercontent.com/31318730/85798825-30890c80-b70c-11ea-8878-4ea1a799cdaf.png)

**Environment**
 - OS: Windows 10
 - Chrome
 - Version 1.27.1
 - Desktop
- Using Nuxt

**Comment by samarsault** [comment]
For anyone facing this problem, checkout this
https://github.com/scrumpy/tiptap/blob/92eb2c61ccf2780c51212724fd879ba343436bd3/packages/tiptap-extensions/src/nodes/CodeBlockHighlight.js#L11-L13

If anyone is facing the problem, try the following
```js
new CodeBlockHighlight({
    options: {
        languages: { javascript, css, xml, python, java, clike },
    }
})
```

**Comment by nezzard** [comment]
Code is not highlighting 
    "tiptap": "^1.27.1",
    "tiptap-extensions": "^1.29.1",

**Comment by anirudhRowjee** [comment]
I am facing the same issue. I even copy-pasted the `index.vue` file directly from the source and ran it as a separate component. no joy.

**Comment by egorovli** [comment]
After debugging the source of `tiptap-extensions`, I've found that the actual error has nothing to do with how you pass languages. It still should be:

```js
new CodeBlockHighlight({
  languages: { javascript, css, xml }
})
```

The error however is the following:

![image](https://user-images.githubusercontent.com/5165202/86102650-76572380-bac4-11ea-8cea-b6f0b15678e2.png)

Apparently it comes from version mismatch of the `highlight.js`. In my `package.json` I have `^10.1.1`, but `lowlight` depends on version `9.16.2`. Fixed the issue by setting the version of `highlight.js` to `^9.16.2` in the `package.json`:

```json5
{
  // ...
  "dependencies": {
    "highlight.js": "^9.16.2"
  },
  // ...
}
```
